src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -434,10 +435,18 @@ protected void registerRecipes(Consumer<IFinishedRecipe> consumer)
 		CombinationRecipeBuilder.of(MSItems.SKAIA_FORK).input(MSItems.FORK).and().input(MSBlocks.CHESSBOARD).build(consumer);
 		CombinationRecipeBuilder.of(MSItems.SPORK).input(MSItems.FORK).or().input(MSItems.WOODEN_SPOON).build(consumer);
 		CombinationRecipeBuilder.of(MSItems.GOLDEN_SPORK).input(MSItems.SPORK).or().input(Items.GOLD_INGOT).build(consumer);
+		CombinationRecipeBuilder.of(MSItems.BIDENT).input(MSItems.BI_DYE).and().input(Items.TRIDENT).build(consumer);
 		
 		CombinationRecipeBuilder.of(MSItems.CAT_CLAWS_SHEATHED).input(Items.IRON_BARS).and().input(Items.LEATHER).build(consumer);
         CombinationRecipeBuilder.of(MSItems.CIGARETTE_LANCE).input(Items.IRON_SWORD).and().input(Tags.Items.RODS_WOODEN).build(consumer);
 		
+        CombinationRecipeBuilder.of(MSBlocks.LAPTOP).input(MSBlocks.COMPUTER).and().namedInput(MSItems.BATTERY).build(consumer);
+		CombinationRecipeBuilder.of(MSBlocks.LAPTOP).input(MSBlocks.COMPUTER).or().namedInput(Items.BOOK).build(consumer);
```

Comment:
With what alchemy recipes are based on, it does not make sense if you combine item a & b to make item c, that you would also be able to combine item a | d to make item c. I'd suggest using the same operator for the two of them

Reply:
I was making these recipes while working with the supposed complexity of the items in mind. It is not incredibly important to me but I think the recipe types match the typical use of those alchemy types: 

"an && alchemy result will often demonstrate the functionality of both components – e.g. Dave's iShades can be a phone and a rad pair of shades – while an || alchemy result will typically have the functionality of only one of the components, but the form of the other – e.g. the Hammerhead Pogo Ride" https://mspaintadventures.fandom.com/wiki/Alchemiter

Reply:
Perhaps, but I don't think that's enough in this scenario since those two recipes form a pretty clear contradiction to how the alchemy system works (were I able to convey what the problem/contradiction is?)